common/azure-common/src/main/java/com/azure/common/http/policy/RetryPolicy.java

```diff
@@ -20,6 +20,7 @@
     private static final int DEFAULT_MAX_RETRIES = 3;
     private static final int DEFAULT_DELAY = 0;
     private static final ChronoUnit DEFAULT_TIME_UNIT = ChronoUnit.MILLIS;
+    private static final String RETRY_AFTER_MS_HEADER = "retry-after-ms";
```

Comment:
Just FYI, we use more headers in [.NET](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/Policies/RetryPolicy.cs#L15) to identify a response from the service when it is related to Retry Policy 
You might want to consider them too.

FYI @pakrym

Reply:
Thanks for the information, I will keep the other headers in mind for future implementation.